src/com/game/controller/Game.java

```diff
@@ -38,14 +38,14 @@ public Game() {
 
     }
     //This should be called by the client to start a new game.
-    public void start(){
+    public void start() throws IOException {
```

Comment:
Same for all these IO Exceptions. These should be managed in the method

Reply:
Fixed

Reply:
Fixed